builder/index.js

```diff
@@ -1,4 +1,4 @@
-const Queue = require('./rabbitmq')
+const Queue = require('./queue')
```

Comment:
Oh I was actually refering to the name of the variable (`Queue`) which is inconsistent with the other names (`execFile`) because it looks like it's in CamelCase while the other are in pascalCase.

I don't know if that's voluntary nor what are the rules behind it, or if it's a mistake.

Reply:
It is voluntary because Queue is a class and then I use the variable "queue" to store an instance of the class.